### PR TITLE
[ONT-791] Cli add gen multi signature address cmd

### PR DESCRIPTION
**Pull request by gasby88**
signed-off-by:wangbing@onchain.com

### CODE REVIEW

File: cmd/account_cmd.go [hunk 1]
```diff
@@ -543,3 +555,64 @@ func importWIF(filepath string, wallet account.Client) error {
...
+	walletPath := ctx.String(utils.GetFlagName(utils.WalletFileFlag))
+	if walletPath != "" {
+		wallet, err = account.Open(walletPath)
+		if err != nil {
+			return fmt.Errorf("Cannot open wallet:%s", walletPath)
+		}
+	}
+
+	pks := strings.Split(pkstr, ",")
+	pkSize := len(pks)
```
Comment (Arbio5zt) [inline cmd/account_cmd.go]: move line 599~ 605 here to check the value first before make.

**Review by Arbio5zt — CHANGES_REQUESTED** [review]

**Comment by laizy** [comment]
pending this for one or two days after intergration branch is ready. Since it has some changes in that branch.

File: cmd/account.go [hunk 2]
```diff
@@ -30,6 +30,8 @@ import (
 	"strings"
 )
 
+const MAX_PUBLIC_KEY_SIZE = 24
```
Comment (laizy) [inline cmd/account.go]: neo's max size is 1024, and defined in common/constants module.

File: cmd/utils/flags.go [hunk 3]
```diff
@@ -227,6 +227,14 @@ var (
 		Name:  "wif",
 		Usage: "Import WIF keys from the source file specified by --source option",
 	}
+	AccountMultiMFlag = cli.UintFlag{
+		Name:  "m",
+		Usage: "M of multi signature address. m must > 0 and < 24, and m must <= number of pub key",
```
Comment (laizy) [inline cmd/utils/flags.go]: update to `1024`

File: cmd/utils/flags.go [hunk 4]
```diff
@@ -227,6 +227,14 @@ var (
...
+	}
+	AccountMultiPubKeyFlag = cli.StringFlag{
+		Name:  "pubkey",
+		Usage: "Pub key list of multi address, split pub key with `,`. Number of pub key must > 0 and <= 24",
```
Comment (laizy) [inline cmd/utils/flags.go]: ditto

**Review by laizy — CHANGES_REQUESTED** [review]

**Review by laizy — APPROVED** [review]

**Pull request merged**